ludwig/features/numerical_feature.py

```diff
@@ -47,12 +47,32 @@ def __init__(self, feature):
 
     preprocessing_defaults = {
         'missing_value_strategy': FILL_WITH_CONST,
-        'fill_value': 0
+        'fill_value': 0,
+        'normalization_strategy': None
     }
 
     @staticmethod
     def get_feature_meta(column, preprocessing_parameters):
-        return {}
+        if preprocessing_parameters['normalization_strategy'] is not None:
+            if preprocessing_parameters['normalization_strategy']=='zscore':
+                return {'stats': {
```

Comment:
stats is probably redundant, you can put the two values directly in the returned dictionary (here and everywhere else obviously)

Reply:
Changed it!